common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
```

Comment:
We have separate function deploy_common_services() at line 159 which will call this deploy_common_service() to deploy marathon-lb  and wso2-shared-dbs.